### PR TITLE
jar: added jar listing options examples

### PR DESCRIPTION
**Pull request by eranharel**
Added `jar tf` and `jar tvf` listing examples

----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/jar.md [hunk 1]
```diff
@@ -5,3 +5,11 @@
 - Unzip .jar/.war file to the current directory:
 
 `jar -xvf *.jar`
+
+- List a .jar/.war file content
+
+`jar tf /path/to/my.jar`
```
Comment (pxgamer) [inline pages/common/jar.md]: Add token syntax for this: `{{path/to/file.jar}}`

File: pages/common/jar.md [hunk 2]
```diff
@@ -5,3 +5,11 @@
 - Unzip .jar/.war file to the current directory:
 
 `jar -xvf *.jar`
+
+- List a .jar/.war file content
```
Comment (pxgamer) [inline pages/common/jar.md]: Add a colon to the end of the description.

File: pages/common/jar.md [hunk 3]
```diff
@@ -5,3 +5,11 @@
...
+
+`jar tf /path/to/my.jar`
+
+- List a .jar/.war file content with verbose output
```
Comment (pxgamer) [inline pages/common/jar.md]: Add a colon to the end of the description here.

File: pages/common/jar.md [hunk 4]
```diff
@@ -5,3 +5,11 @@
...
+
+`jar tvf /path/to/my.jar`
```
Comment (pxgamer) [inline pages/common/jar.md]: Add token syntax for this: `{{path/to/file.jar}}`

File: pages/common/jar.md [hunk 5]
```diff
@@ -5,3 +5,11 @@
 - Unzip .jar/.war file to the current directory:
 
 `jar -xvf *.jar`
+
+- List a .jar/.war file content:
+
+`jar tf {{/path/to/my.jar}}`
```
Comment (pxgamer) [inline pages/common/jar.md]: Sorry, without the prefixing slash for both of these 🙂

File: pages/common/jar.md [hunk 6]
```diff
@@ -5,3 +5,11 @@
 - Unzip .jar/.war file to the current directory:
 
 `jar -xvf *.jar`
```
Comment (agnivade) [inline pages/common/jar.md]: While we are here, let us change this to {{file.jar}}

File: pages/common/jar.md [hunk 7]
```diff
@@ -5,3 +5,11 @@
...
+
+- List a .jar/.war file content:
+
+`jar tf {{path/to/my.jar}}`
```
Comment (agnivade) [inline pages/common/jar.md]: Change to file.jar for consistency with other pages.

File: pages/common/jar.md [hunk 8]
```diff
@@ -5,3 +5,11 @@
...
+
+- List a .jar/.war file content with verbose output:
+
+`jar tvf {{path/to/my.jar}}`
```
Comment (agnivade) [inline pages/common/jar.md]: same as above

**Pull request merged**